test/api/search_controller_test.rb

```diff
@@ -2,45 +2,70 @@
 require 'user_tokens_helper'
 require 'bulk_download_helper'
 require 'test_helper'
+require 'big_query_helper'
 
 class SearchControllerTest < ActionDispatch::IntegrationTest
   include Devise::Test::IntegrationHelpers
   include Requests::JsonHelpers
   include Requests::HttpHelpers
   include Minitest::Hooks
   include ::TestInstrumentor
+  include ::SelfCleaningSuite
 
-  HOMO_SAPIENS_FILTER = { id: 'NCBITaxon_9606', name: 'Homo sapiens' }
-  NO_DISEASE_FILTER = { id: 'MONDO_0000001', name: 'disease or disorder' }
+  HOMO_SAPIENS_FILTER = { id: 'NCBITaxon_9606', name: 'Homo sapiens' }.freeze
+  NO_DISEASE_FILTER = { id: 'MONDO_0000001', name: 'disease or disorder' }.freeze
 
   # shorthand accessors
   FACET_DELIM = Api::V1::SearchController::FACET_DELIMITER
   FILTER_DELIM = Api::V1::SearchController::FILTER_DELIMITER
 
+  before(:all) do
+    @random_seed = SecureRandom.uuid
```

Comment:
Could you help me understand why `@random_seed` is used in this test but not others in this PR?